compiler/src/iree/compiler/Dialect/Stream/Transforms/Passes.td

```diff
@@ -165,6 +165,33 @@ def MaterializeCopyOnWritePass :
   ];
 }
 
+def MaterializeEncodeOpsPass :
```

Comment:
```suggestion
def MaterializeEncodingsPass :
```
(it doesn't materialize encode ops, but materializes the encoding executables that implement them and erases them)

Reply:
(I named it `MaterializeEncodingsPass` during the prototype, and I was confused that which one is better during the PR implementation. Thank you for making it clear, what you said makes sense to me.)